Shared/Features/Freaks/View/MultipleSelectionRow.swift

```diff
@@ -0,0 +1,23 @@
+import SwiftUI
+
+struct MultipleSelectionRow: View {
+    @ObservedObject var viewModel = FreaksViewModel()
```

Comment:
You should user @ObservedObject only if you pass this class from another view. If you want to instantiate the view model in this view you should use @StateObject.